.github/workflows/mirror_protoc_release.yml

```diff
@@ -0,0 +1,19 @@
+name: Mirror Releases
+on:
+  # Trigger manually in the UI
+  workflow_dispatch:
+  # Trigger daily at 06:10 UTC
+  schedule:
+    - cron: "10 6 * * *"
```

Comment:
I'm missing what prevents this script to create a PR every day.

Reply:
The create-pr-action only creates a PR if there's a change. I've already been operating this at https://github.com/alexeagle/toolchains_protoc/actions/workflows/mirror_protoc_release.yml and you can see it sends PRs correctly whenever there's a protoc release: https://github.com/alexeagle/toolchains_protoc/pulls?q=is%3Apr+author%3Aapp%2Fgithub-actions+

Reply:
added a comment above the schedule: block